nixos/doc/manual/release-notes/rl-1903.xml

```diff
@@ -645,6 +645,9 @@
        This may break some older applications that still rely on those symbols.
        An upgrade guide can be found <link xlink:href="https://www.open-mpi.org/faq/?category=mpi-removed">here</link>.
      </para>
+    <para>
+     The nginx now relies on OpenSSL 1.1 and supports TLS 1.3 by default. You can change the supported protocols using <xref linkend="opt-services.nginx.sslProtocols"/>
```

Comment:
Wordsmithing: "The nginx package now requires OpenSSL, and supports TLS 1.3 by default."

Reply:
It always required OpenSSL, we just switched from 1.0 to 1.1 branch.

Reply:
I was more referring to the lack of the word "package" to qualify "nginx" to help it read easier, and placing emphasis on the "requirement" part. Yes, the full text should be:

> The nginx package now requires OpenSSL 1.1, and supports TLS 1.3 by default.